<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Grid Display</string>
   </property>
   <widget class="QRadioButton" name="optGridOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optGridFast">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optGridPretty">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pretty</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Folder</string>
   </property>
   <widget class="QLineEdit" name="txtDataFolder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseDataFolder">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
